packages/react-dom/src/events/ReactBrowserEventEmitter.js

```diff
@@ -172,11 +181,11 @@ export function isListeningToAllDependencies(
   registrationName: string,
   mountAt: Document | Element,
 ) {
-  const isListening = getListeningForDocument(mountAt);
+  const isListening = getListenerTrackingFor(mountAt);
   const dependencies = registrationNameDependencies[registrationName];
   for (let i = 0; i < dependencies.length; i++) {
     const dependency = dependencies[i];
-    if (!(isListening.hasOwnProperty(dependency) && isListening[dependency])) {
```

Comment:
I'm going to revert this change. It isn't related to this work and should be done in a separate PR.